charts/consul/templates/server-statefulset.yaml

```diff
@@ -124,7 +124,9 @@ spec:
       serviceAccountName: {{ template "consul.fullname" . }}-server
       {{- if not .Values.global.openshift.enabled }}
       securityContext:
-        {{- toYaml .Values.server.securityContext | nindent 8 }}
+        {{- $tmp := dict -}}
```

Comment:
`merge` will fill `$tmp` with the merged results, if we don't setup a tmp variable it will overwrite the original data (`.Values.global.podSecurityStandards.securityContext`) which gets used elsewhere.